src/common/hook/useQueue/utils/publish.tsx

```diff
@@ -100,14 +113,19 @@ export const groupDocumentsIntoJobs = async (
   let nonce = currentNonce;
 
   const jobs: PublishingJob[] = [];
-
   // Process all verifiable documents with document store first
-  documentStoreAddresses.forEach((contractAddress) => {
+  for (const contractAddress of documentStoreAddresses) {
```

Comment:
was wondering why switch to `for .. of` from `fromEach`

Reply:
cause wrapDocumentsV3 is async, and require the wrapped doc to be pushed into the jobs array before returning. thus it is easier to implement async/await using for..of compare to forEach

Reply:
hmm was `wrapDocumentsV2` sync previously? i was under the impression it was always async (ah its not, just checked).